torch/_inductor/fx_passes/efficient_conv_bn_eval.py

```diff
@@ -70,6 +75,160 @@ def efficient_conv_bn_eval(
     return output
 
 
+def efficient_conv_bn_fn_eval(
```

Comment:
Quick question: does this shadow the original `efficient_conv_bn_fn_eval` function defined above?

Reply:
It is copied and modified version of the `efficient_conv_bn_eval`, note that my copied version has a `fn` in the name so naming won't conflict.

Reply:
Got it.

Reply:
@JackCaoG I suggest suffixing `_fn` after the function name. Adding `_fn` in the function name is quite confusing.

Reply:
let me renamed it to something like `efficient_conv_bn_eval_decomposed` to make it less confusing